routing/URI.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Routing;
+
+/**
+ * URI.
+ */
+interface URI
+{
+
+    /**
+     * Returns an array of URI subparts.
+     *
+     * @return string[]
+     */
+    public function parted(): array;
+
+    /**
+     * Returns number of subparts in the URI.
+     *
+     * @return int
+     */
+    public function count(): int;
```

Comment:
No demand for this method right know. However, it will probably be used later. If it is useless then it can be deleted.